assets/chat/js/chat.js

```diff
@@ -1257,12 +1258,19 @@ class Chat {
       if (!this.backlogloading) {
         const retryMilli = Math.floor(Math.random() * 30000) + 4000;
         setTimeout(() => window.location.reload(true), retryMilli);
+
         MessageBuilder.broadcast(
-          `Restart incoming in ${Math.round(retryMilli / 1000)} seconds ...`,
+          `Restart incoming in ${Math.round(retryMilli / 1000)} seconds...`,
+          new ChatUser(),
```

Comment:
Would you mind passing in an object with the system user's properties to the constructor here?

Reply:
Done!